react/features/reactions/components/web/ReactionEmoji.tsx

```diff
@@ -79,18 +77,15 @@ class ReactionEmoji extends Component<Props, State> {
         return (
             <div
                 className = { `reaction-emoji reaction-${index}` }
-                id = { uid }>
+                id = { `${uid}` }>
```

Comment:
Hmm, isn't this already converted to string?

Reply:
No, it was number, but I changed the interface to be string